intake/source/csv.py

```diff
@@ -54,12 +54,19 @@ def __init__(self, urlpath, csv_kwargs=None, metadata=None,
     def _set_pattern_columns(self, path_column):
         """Get a column of values for each field in pattern
         """
+        try:
+            from pandas.api.types import CategoricalDtype
+            _HAS_CDT = True
```

Comment:
This is a Pandas version thing? Maybe nice to have a clarifying comment.

Reply:
Right it is >=0.21.0 https://pandas.pydata.org/pandas-docs/stable/generated/pandas.api.types.CategoricalDtype.html#pandas.api.types.CategoricalDtype

Reply:
fixed